packages/eds-core-react/src/components/Datepicker/utils/useGetLocale.ts

```diff
@@ -0,0 +1,13 @@
+import { useLocale } from 'react-aria'
+
+export const useGetLocale = (locale?: string) => {
+  const { locale: externalLocale } = useLocale()
+  // react-aria defaults to navigator.language if no locale is provided. If these are equal, we override by using the system default locale
+  const defaultLocale = navigator.language
```

Comment:
I think there needs to be an existence check for navigator since it will be undefined in nextjs